addons/mouse-pos/addon.json

```diff
@@ -21,6 +21,6 @@
       "matches": ["projects"]
     }
   ],
-  "tags": ["editor", "projectPlayer", "recommended"],
+  "tags": ["editor", "recommended"],
```

Comment:
Same here. You should consider that someone might add website support in the future.
```suggestion
  "tags": ["player", "editor", "recommended"],
```

Reply:
According to the [addon manifest docs](https://scratchaddons.com/docs/reference/addon-manifest/#required-factual-tags) addons can only have one top-level category tag.

Reply:
> According to the [addon manifest docs](https://scratchaddons.com/docs/reference/addon-manifest/#required-factual-tags) addons can only have one top-level category tag.

Really? Oh, sorry then.

Reply:
Clone counter still needs the `player` tag.